packages/cli/commands/upload.js

```diff
@@ -204,45 +253,30 @@ exports.handler = async options => {
       absoluteSrcPath,
       options.convertFields
     );
-    uploadFolder(
+
+    const noModules = filePaths.filter(path => !path.includes('/modules/'));
+    await uploadFiles(
+      accountId,
+      absoluteSrcPath,
+      dest,
+      {
+        mode,
+      },
+      options,
+      noModules
+    );
+
+    const onlyModules = filePaths.filter(path => path.includes('/modules/'));
```

Comment:
Not a blocker, but is there any easy way to build `onlyModules` and `noModules` without having to iterate `filePaths` twice?

Reply:
Lodash has a `partition` that I think would work here